### PR TITLE
[9.0] base_geoengine - restricted extents

### PR DESCRIPTION
**Pull request by yvaucher**
Depends on
- [x] #60 
- [x] #72

To work properly with projection it needs #79 

Add restricted extent option on view

### CODE REVIEW

**Comment by lmignon** [comment]
@yvaucher a rebase is needed.

**Comment by yvaucher** [comment]
Rebased

File: base_geoengine/geo_model.py [hunk 1]
```diff
@@ -178,7 +163,10 @@ def get_edit_info_for_geo_column(self, cursor, uid, column, context=None):
         res['edit_raster'] = raster_obj.read(
             cursor, uid, raster_id[0], context=context)
         res['geo_type'] = field.geo_type
-        res['srid'] = field.srid
+        # get srid from _columns as srid is wrong in _fields
+        res['srid'] = self._columns.get(column)._srid
```
Comment (lmignon) [inline base_geoengine/geo_model.py]: If the field is not stored, `self._column.get(column)` will return None

Comment (lmignon) [inline base_geoengine/geo_model.py]: @yvaucher see https://travis-ci.org/OCA/geospatial/jobs/120637898#L481 That's why you can't get the srid from the column...

Comment (yvaucher) [inline base_geoengine/geo_model.py]: Better to discuss this here #79

**Comment by lmignon** [comment]
@yvaucher Usefull new functionnality :+1:  Code review + Functional tests

**Comment by guewen** [comment]
:+1:

**Pull request merged**